Training/ObjectDetection.py

```diff
@@ -7,29 +7,34 @@
 import random
 import matplotlib.pyplot as plt
 import torchvision.transforms as T
-from QA.Normalization.Colour import ColourNorm
+from QA.Normalization.Colour import ColourNorm_old
 import albumentations as A
-
+from Utils.ObjectDetectionTools import collate_fn
 import torch
 from pytorch_lightning.callbacks import ModelCheckpoint
 from pytorch_lightning.loggers import TensorBoardLogger
 from pytorch_lightning import Trainer, seed_everything
 from Dataloader.ObjectDetection import MFDataModule
 from Model.MaskRCNN import MaskFRCNN
+from sklearn.model_selection import train_test_split
+
+import resource
+rlimit = resource.getrlimit(resource.RLIMIT_NOFILE)
+resource.setrlimit(resource.RLIMIT_NOFILE, (4096, rlimit[1]))
 
 config = toml.load(sys.argv[1])
-n_gpus = torch.cuda.device_count()
+n_gpus = len(config['MODEL']['GPU_ID'])
 
 augmentation = A.Compose([
     A.RandomCrop(width=config['DATA']['dim'][0], height=config['DATA']['dim'][1]),
     A.HorizontalFlip(p=config['AUGMENTATION']['horizontalflip']),
-    #A.RandomBrightnessContrast(p=config['AUGMENTATION']['randombrightnesscontrast']),
+    A.RandomBrightnessContrast(p=config['AUGMENTATION']['randombrightnesscontrast']),
```

Comment:
Why use albumentations rather than torchvision?

Reply:
@SZY1234567 please advise

Reply:
albumentations can do transforms for the image and the paired mask